Makefile

```diff
@@ -214,9 +214,13 @@ integration_all: $(MINISHIFT_BINARY)
 	--run-before-feature="$(RUN_BEFORE_FEATURE)" --test-with-specified-shell="$(TEST_WITH_SPECIFIED_SHELL)" --copy-oc-from="$(COPY_OC_FROM)" $(GODOG_OPTS)
 
 .PHONY: integration_pr ## Run integration tests for pull request
-integration_pr: GODOG_OPTS = --tags=~coolstore\&\&~addon-xpaas
+integration_pr: GODOG_OPTS = --tags=cmd-openshift
 integration_pr: $(MINISHIFT_BINARY)
 	mkdir -p $(INTEGRATION_TEST_DIR)
+	echo "HTTP_PROXY=$(HTTP_PROXY) $(origin HTTP_PROXY)"
+	echo "HTTPS_PROXY=$(HTTPS_PROXY) $(origin HTTPS_PROXY)"
+	echo "MINISHIFT_HTTP_PROXY=$(MINISHIFT_HTTP_PROXY) $(origin MINISHIFT_HTTP_PROXY)"
+	echo "MINISHIFT_HTTPS_PROXY=$(MINISHIFT_HTTPS_PROXY) $(origin MINISHIFT_HTTPS_PROXY)"
```

Comment:
@agajdosi what all those echoes for?

Reply:
@praveenkumar oh, I have missed those, was just for debugging purposes, going to remove it now. thanks for noticing.

Reply:
also line 217 needs fix